src/util/PopulationsMap.ts

```diff
@@ -439,19 +442,128 @@ export const mapExistingTestCase = (existingTestCasePC, group: Group) => {
     group?.measureObservations?.length > 0
   ) {
     if (isBooleanOrPatientBasis) {
-      return addDefaultObservationsForBooleanAndPatientBasedTestcases(
+      return addDefaultObservationsForPatientBasedTestcases(
         existingTestCasePC,
         group
       );
     } else {
-      // need to implement for non-boolean and episode based measures
-      return { ...existingTestCasePC };
+      return addDefaultObservationsForEpisodeBasedTestcases(
+        existingTestCasePC,
+        group
+      );
+    }
+  }
+  return { ...existingTestCasePC };
+};
+
+const countObservations = (
+  expectedRelatedPopulationValue,
+  expectedRelatedPopulationExclusionValue
+) => {
+  if (
+    expectedRelatedPopulationValue &&
+    expectedRelatedPopulationExclusionValue
+  ) {
+    const observationsCount = Math.max(
+      expectedRelatedPopulationValue - expectedRelatedPopulationExclusionValue,
+      0
```

Comment:
Can you check if the default value can be 0 ?

Reply:
Here I am not assigning any values I am just getting the count of how many observations need to be added. so in this case this is situation when : denominator - denominatorExclusion value is -ve value. In that case I am setting the count of observations to be added is equal to 0 (therefore no default observations would be added )

Reply:
Cool